Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "relm4-icons"
 authors = ["Aaron Erhardt <aaron.erhardt@t-online.de>"]
-version = "0.6.0"
+version = "0.6.1"
```

Something like this?

```suggestion
version = "0.7.0"
```